front/temporal/labs/utils/gong.ts

```diff
@@ -141,7 +141,10 @@ export async function retrieveGongTranscriptContent(
 } | null> {
   if (!transcriptsConfiguration || !transcriptsConfiguration.connectionId) {
     localLogger.error(
-      {},
+      {
+        fileId,
```

Comment:
`nit` but would be better to create a child logger with those to avoid duplicating.